format-specs/geoparquet.md

```diff
@@ -13,7 +12,7 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 ## Version
 
-This is version 0.5.0-dev of the GeoParquet specification.
+This is version 1.0.0-dev of the GeoParquet specification.
```

Comment:
We can adopt a different convention, but I think of the `-dev` modifier as being applied to the next release that we "want" - so typically a minor release instead of a patch release.  We could have this be any placeholder really.  It could be `unreleased` even.